.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/43799727/144861040-69b4dfe2-f0d1-4bb1-9041-fb6cf8a81efa.png)

👍 Great job! However, some changes are required.

See inline comments for required changes.